pkg/splunk/enterprise/util.go

```diff
@@ -913,16 +913,23 @@ func changePhaseInfo(desiredReplicas int32, appSrc string, appSrcDeployStatus ma
 				continue
 			}
 
+			// CSPL-1570: If this is the first time we are scaling up from replicas=1, then set the AuxPhaseInfo to Install complete.
```

Comment:
General question - why don't we always keep the `AuxPhaseInfo` up-to-date, irrespective of whether or not we are scaling up? If we have this already in place, we could just skip over the replicas which already have the status as `AppPkgInstallComplete` ?

Reply:
As I have replied below in other comment, `AuxPhaseInfo` is always up-to-date. This was a special and only case left that was yet to be handled.